Theme.scss

```diff
@@ -7517,6 +7536,18 @@ a.btn,
     background-color: #bd4a29 !important;
 }
 
+.user-status-message-wrapper div {
+    color: white !important;
+}
+
+.user-status-message-wrapper div :hover {
+    color: #0000EE !important;
+}
```

Comment:
I dont see the need for this, there's already a blue on hover.
```suggestion
```

Reply:
https://github.com/DarkThemeHub/GithubDarkTheme/pull/173#discussion_r444558006

same for this